src/reqwest.js

```diff
@@ -590,6 +591,7 @@
   // .ajax.compat(options, callback)
   reqwest.compat = function (o, fn) {
     if (o) {
+      o = Object.create(o)
```

Comment:
I'm not clear how this fixes the compat issue. and this is likely not going to work in some of the old supported browsers...